File: aws-lambda-functions/nodejs/Invoke_Lambda.mjs
import { LambdaClient, InvokeCommand } from "@aws-sdk/client-lambda";

const lambdaClient = new LambdaClient({});
const TARGET_FUNCTION = "arn:aws:lambda:eu-central-1:191880149831:function:invocation-Lambda";

export const handler = async (event, context) => {
    try {
        const payload = JSON.stringify({ "message": "Hello from Node Lambda" });

        const command = new InvokeCommand({
            FunctionName: TARGET_FUNCTION,
            InvocationType: "Event", 
            Payload: new TextEncoder().encode(payload)
        });

        await lambdaClient.send(command);

        console.log("Invocation sent (async)");
        return "ok";
        
    } catch (error) {
        console.error("Error: " + error.message);
        return "error";
    }
};

File: aws-lambda-functions/python/Invoke_Lambda.py
import boto3
import json

lambda_client = boto3.client('lambda')
TARGET_FUNCTION = "arn:aws:lambda:eu-central-1:191880149831:function:invocation-Lambda"

def lambda_handler(event, context):
    try:
        payload_data = {"message": "Hello from Python Lambda"}
        
        lambda_client.invoke(
            FunctionName=TARGET_FUNCTION,
            InvocationType='Event',  
            Payload=json.dumps(payload_data)
        )

        print("Invocation sent (async)")
        return "ok"
        
    except Exception as e:
        print(f"Error: {str(e)}")
        return "error"